Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -55,6 +55,13 @@ open class NavigationMapView: UIView {
      If `true` and there're multiple routes to choose, the alternative route lines would display the congestion levels at different colors, similar to the main route. To customize the congestion colors that represent different congestion levels, override the `alternativeTrafficUnknownColor`, `alternativeTrafficLowColor`, `alternativeTrafficModerateColor`, `alternativeTrafficHeavyColor`, `alternativeTrafficSevereColor` property for the `NavigationMapView.appearance()`.
      */
     public var showsCongestionForAlternativeRoutes: Bool = false
+    
+    /**
+     Controls whether to show fading gradient congestion color on route lines. Defaults to `false`.
+     
+     If `true`, the congestion level in the route line will be shown as fading gradient color instead of abrupt and steep change.
+     */
+    public var showFadingCongestionColor: Bool = false
```

Comment:
“showFadingCongestionColor” sounds like a method name; “show**s**FadingCongestionColor” would be more appropriate for a property name. Better yet, “crossfadesCongestionSegments” would be more descriptive and avoid confusion with the vanishing (fading out) route line feature.

Reply:
Thanks for the suggestion, I'll update the name.